app/controllers/nodes_controller.rb

```diff
@@ -114,6 +113,26 @@ def tree
   end
 
   private
+
+  def set_evidence_default_columns
+    default_field_names = ['Title', 'Created', 'Updated'].freeze
+    extra_field_names = ['Created by'].freeze
```

Comment:
Let's make this consistent with `Issues::Evidence#index` table: Label and Title, moving Created and Updated down to `extra_`

Reply:
Done